src/hooks/twilio.ts

```diff
@@ -157,13 +163,13 @@ export const useParticipantState = ({
 export interface UseVideoRoomStateProps {
   userId?: string;
   roomName?: string;
-  showVideoByDefault?: boolean;
+  defaultActiveParticipant?: boolean;
```

Comment:
Can we word this in a way that flows better like it used to? show

Reply:
Yep, I tried to follow the naming patterns set in `useParticipantState`.

![image](https://user-images.githubusercontent.com/63194656/121222740-7a242600-c88f-11eb-85a9-9cb5e9190e0b.png)

Renamed in https://github.com/sparkletown/sparkle/pull/1513/commits/db7071fb8e926299f9a82a9eface65db11ab5c1b